Src/AutoFixture/Fixture.cs

```diff
@@ -170,6 +170,7 @@ public Fixture(ISpecimenBuilder engine, MultipleRelay multiple)
             this.UpdateGraphAndSetupAdapters(newGraph, Enumerable.Empty<ISpecimenBuilderTransformation>());
 
             this.Behaviors.Add(new ThrowingRecursionBehavior());
+            this.Behaviors.Add(new ReadonlyCollectionPropertiesBehavior());
```

Comment:
I have removed this as default behavior for now. I'd agree that this would be a breaking change, and is perhaps a behavior that should be explicitly requested.